addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/handler/SamsungTvHandler.java

```diff
@@ -357,10 +247,13 @@ private void checkCreateManualConnection() {
                     stopService(service);
                 }
             } else {
-                logger.trace("One or more services are already registered, not checking for new ones");
+                logger.debug("One or more services are already registered, not checking for manual new ones");
+                logAvailableServices();
             }
         } catch (RuntimeException e) {
-            logger.warn("Catching all exceptions because otherwise the thread would silently fail", e);
+            logger.warn(
+                    "Catching all exceptions because otherwise the checkCreateManualConnection thread would silently fail",
```

Comment:
This can be a comment in the code, but should not be the message of the warning.